src/pages/venues/[id]/index.tsx

```diff
@@ -403,22 +403,6 @@ export function VenuePageContent() {
                 ]}
               />
             )}
-            <InfoBlock
-              headingLevel="h3"
-              icon={<IconLocation aria-hidden="true" />}
-              name={t("block.location.label")}
-              contents={[
-                <InfoBlock.List
-                  key="address"
-                  items={[name, streetAddress, addressLocality]}
-                />,
-                <InfoBlock.Link
-                  key="map-link"
-                  href={{ pathname: `/venues/[id]/map`, query: { id } }}
-                  label={t("link.map.label")}
```

Comment:
Could this label be removed as well?

Reply:
i think it was used somewhere else, i ll recheck

Reply:
no, it was not, removed